src/modules/tests/test-data/test-data.selector.ts

```diff
@@ -63,7 +63,7 @@ export const hasEyesightTestBeenCompleted = (data: TestData) => data.eyesightTes
 export const hasEyesightTestGotSeriousFault = (data: CatBUniqueTypes.TestData) => data.eyesightTest.seriousFault;
 
 export const hasLegalRequirementBeenCompleted = (
-  data: CatBUniqueTypes.TestRequirements, legalRequirement: LegalRequirements) => {
+  data: CatBUniqueTypes.TestRequirements, legalRequirement: LegalRequirements | CatBeLegalRequirements) => {
```

Comment:
eeeek I think I missed the boat on this one, but I don't think separate, category specific enums/labels was a good idea here, having this union type defeats the whole point! The purpose of the Enum is to ensure that the legal requirement being operated upon is a valid legal requirement. The schema is what ensures that the legal requirement is valid for a specific category.
I think we should get revert this unless I've missed a compelling reason to keep it, sorry @gittins7!

Reply:
So this function should deal with only cat-b specific legal requirements

Reply:
ah ok that makes sense, so we create a new `src/modules/tests/test-data/test-data.selector.cat-be.ts` file that exposes the same functions but are typed to `CatBEUniqueTypes`..
We can still just have one set of constants including the enums